content/1_General/USACO_FAQs.mdx

```diff
@@ -238,7 +239,25 @@ or by googling. Answers to additional FAQs can be found
 You can also ask a question by submitting the "Contact Us" form on the
 bottom-left of this page, or on the [USACO Forum](https://forum.usaco.guide/).
 
+## Q: Is collaborative work allowed?
+
+Collaborating with others during the contest is strictly prohibited to maintain fairness and integrity. USACO contests are designed to assess individual problem-solving skills, and collaboration provides an unfair advantage. However, outside of the contest environment, students are encouraged to study and prepare together, sharing knowledge and techniques to improve their skills collectively.
+
+## Q: What are the constraints on memory and time for C++, Python, and Java programs?
+In USACO contests, programs written in various programming languages are subject to specific constraints to ensure fair and efficient competition. These constraints include both time limits and memory allocations, which vary slightly depending on the language used.
+Time Constraints: Each input test case is allotted a specific time limit for program execution. This time limit typically ranges around 4 seconds, although it can vary depending on the complexity of the problem and the specific contest. Within this time frame, the program must complete its execution and produce the correct output. Exceeding the time limit results in the program being terminated, leading to a verdict of "Time Limit Exceeded".
+
+- Memory Constraints: Programs are allocated a certain amount of memory for their execution. The allocated memory is used to store variables, data structures, and other program-related information during execution. It's essential for participants to manage memory efficiently to avoid exceeding the allocated memory limit, which would lead to a "Memory Limit Exceeded" verdict.
+Here's a table summarizing the constraints for different programming languages in USACO contests:
+| Programming Language | Time Limit   | Memory Allocation |
+|----------------------|--------------|-------------------|
+| Java                 | Around 4 sec | Around 256 MB    |
+| C++                  | Around 2 sec | Around 256 MB    |
+| Python               | Around 4 sec | Around 256 MB    |
+
+
 ## Closing Thoughts
 
+If you have any questions please contact the USACO contest director [Brian Dean](mailto:bcdean@clemson.edu).
```

Comment:
Please make it clear that you should email the USACO contest directory only if you are experiencing issues with the USACO website (usaco.org), NOT usaco.guide. Open the contact form for an example